cloudify_awssdk/ec2/resources/subnet.py

```diff
@@ -81,14 +82,14 @@ def delete(self, params=None):
         self.logger.debug('Response: %s' % res)
         return res
 
-    def modify_instance_attribute(self, params=None):
+    def modify_subnet_attribute(self, params=None):
         '''
             Modifies an existing AWS EC2 Subnet Attribute.
         '''
         self.logger.debug(
             'Modifying {0} attribute with parameters: {1}'.format(
                 self.type_name, params))
-        res = self.client.modify_instance_attribute(**params)
+        res = self.client.modify_subnet_attribute(**params)
```

Comment:
Yikes. I feel like this was already fixed in another commit. Do you know if we overwrote it at some point?

Reply:
Not sure if it is already fixed but I will keep it and handle the conflicts